src/test/java/core/basesyntax/service/filehandler/FileWriterTest.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax.service.filehandler;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class FileWriterTest {
+    private static final String NONEXISTENT_FILE_PATH = "nonexistent_folder/nonexistent_file.txt";
+    private static final String STORAGE_REPORT_FILE_PATH = "src/test/java/resources/TestFile.csv";
+    private FileWriter writer;
+    private final String reportText = "Test report content";
+
+    @BeforeEach
+    void setUp() {
+        writer = new FileWriter();
+    }
+
+    @Test
+    public void write_WriteToFile_Ok() throws IOException {
```

Comment:
Maybe better to wrap this IOException. A am not sure, that this important in tests.